server.go

```diff
@@ -253,6 +254,7 @@ func newServer(
 		versionNegotiationQueue:   make(chan receivedPacket, 4),
 		invalidTokenQueue:         make(chan rejectedPacket, 4),
 		connectionRefusedQueue:    make(chan rejectedPacket, 4),
+		retryQueue:                make(chan rejectedPacket, 8),
```

Comment:
Any reason for the queue lengths?

Reply:
This isn't backed by any data, but I'd expect Retries to be sent more frequently than the other types of packets. #4077 will give us more insights at some point.